package.json

```diff
@@ -2,18 +2,27 @@
   "name": "playroom",
   "version": "0.43.1",
   "description": "Design with code, powered by your own component library",
-  "main": "utils/index.js",
-  "types": "utils/index.d.ts",
   "bin": {
     "playroom": "bin/cli.cjs"
   },
+  "main": "./dist/index.js",
+  "module": "./dist/index.mjs",
+  "types": "./dist/index.d.ts",
+  "exports": {
+    ".": {
+      "import": "./dist/index.mjs",
+      "require": "./dist/index.js"
+    },
+    "./package.json": "./package.json"
+  },
   "files": [
     "CHANGELOG.md",
     ".babelrc",
     "images",
     "lib",
     "src",
-    "utils"
+    "utils",
+    "dist"
```

Comment:
Functions in `utils` are still reference internally via relative paths, rather than `import { createUrl } from 'playroom'`, so we still need to include the `utils` directory in the published package.

This may change in the future if (for example) we end up splitting out these utils into a separate package (or the playroom UI itself moves to a separate package.